docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,32 @@ We publish the latest Code Analyzer monthly.
 * Run `sfdx plugins:update` and `sfdx plugins:update --help` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.6.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.6.0) (10-25-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.5.1...v3.6.0)
+
+### Release Summary
+
+* NEW: SFCA v3.x will now publish as `latest`.
```

Comment:
As per the PR at line 29, this should probably be CHANGE, and say "latest and latest-pilot".

Reply:
This would be new to most of our users since they've been on 2.x all this time. Any suggestions to reword to get the effect that we are promoting 3.x from "beta" to "released"? Not sure if "released" is the official term in our case.